src/util.ts

```diff
@@ -26,3 +26,42 @@ export const urlWithContextAsQuery = (url: URL, context: IContext) => {
         });
     return urlWithQuery;
 };
+
+export const contextString = (context: IContext): string => {
+    const { properties = {}, ...fields } = context;
+
+    const sortEntries = (record: Record<string, string>) =>
+        Object.entries(record).sort(([a], [b]) =>
+            a.localeCompare(b, undefined)
+        );
+
+    return JSON.stringify([sortEntries(fields), sortEntries(properties)]);
+};
+
+const sha256 = async (input: string): Promise<string> => {
```

Comment:
I'd use a non-cryptographic hash function suitable for general hash-based lookup e.g. murmurhash that we already use in the node client. It should be faster than cryptography hashing, there's a tiny library we already use elsewhere and it's tested.

Reply:
Also I've just found that we're hashing only the context which is usually quite small so maybe we can start with no hashing in the first version?

Reply:
Yes, we can skip it in this version, but I'd like to have it to prevent storing PII in Unleash storage.

Speed difference is negligible  (~5.5ms for murmur vs ~6.2ms for SHA-256, for medium size context). I don't think it's worth additional weight (https://bundlephobia.com/package/murmurhash3js@3.0.1)